python_modules/dagster/dagster/_core/definitions/schedule_definition.py

```diff
@@ -196,15 +197,7 @@ def __init__(
         self._scheduled_execution_time = check.opt_inst_param(
             scheduled_execution_time, "scheduled_execution_time", datetime
         )
-        self._log_key = (
-            [
-                repository_name,
-                schedule_name,
-                scheduled_execution_time.strftime("%Y%m%d_%H%M%S"),
-            ]
-            if repository_name and schedule_name and scheduled_execution_time
-            else None
-        )
+        self._log_key = log_key
```

Comment:
do we not need to fall back to the old value if a log key is not passed in?

The specific case I am worried out is that you update your user code version / gRPC server, but do not update your daemon (this is not possible in cloud, but is possible in OSS? I've heard mixed things about what guarantees we provide for that case in OSS)

So as I understand it, there is nothing passing in a log key here anymore (since the daemon / grpc callsite is still on the old version), and now we stop logging things since nothing is supplying a log key

am I off-base / would it be hard to fall back to the old format if nothing is passed in?

Reply:
That makes sense to me. I updated the code to fall back to the old format.